OpenRA.Game/Settings.cs

```diff
@@ -20,26 +20,66 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchArgs
```

Comment:
To be more correct, I wanted you to autogenerate the setting code but didn't forsee the problems with these launch args.  I then suggested that the LaunchArgs args was moved away from the settings code (but not removed) as a compromise instead of rewriting that part of the code a second time.  I'm convinced that the settings automation is the best approach, but not convinced at all about the LaunchArgs stuff.